Makefile

```diff
@@ -81,20 +83,19 @@ $(smoketests): k0s
 smoketests: $(smoketests)
 
 .PHONY: check-unit
-check-unit: pkg/assets/zz_generated_offsets_$(TARGET_OS).go
+check-unit: pkg/assets/zz_generated_offsets_$(TARGET_OS).go static/gen_manifests.go
 	go test -race ./pkg/...
 
 .PHONY: clean
 clean:
-	rm -f pkg/assets/zz_generated_offsets_*.go k0s k0s.exe .bins.*stamp bindata*
+	rm -f pkg/assets/zz_generated_offsets_*.go k0s k0s.exe .bins.*stamp bindata* static/gen_manifests.go
 	$(MAKE) -C embedded-bins clean
 
 .PHONY: manifests
 manifests:
 	controller-gen crd paths="./..." output:crd:artifacts:config=static/manifests/helm/CustomResourceDefinition object
 
-.PHONY: bindata-manifests
-bindata-manifests:
+static/gen_manifests.go:
```

Comment:
I think, this should have makefile level dependency on the static files, otherwise bindata wouldn't be regenerated in case if we change something in "static" directory

Reply:
Makes sense. Will add this